cucumber-spring/src/main/java/io/cucumber/spring/CucumberContextConfiguration.java

```diff
@@ -8,7 +8,27 @@
 import java.lang.annotation.Target;
 
 /**
- * Configure an application context class that Cucumber should use
+ * This annotation is used on a configuration class to make the Cucumber aware
+ * of the test configuration. This is to be used in conjunction with
+ * {@code @ContextConfiguration}, {@code @ContextHierarchy} or
+ * {@code @BootstrapWith}. In case of SpringBoot, the configuration class can be
+ * annotated as follows:
+ * <p>
+ * <pre>{@code
```

Comment:
FYI: The magic syntax for multi-line code blocks is `<pre>{@code <the actual code>}</pre>.